tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/base.tsx

```diff
@@ -365,3 +366,23 @@ export const FeatureShortTextInput = (
     />
   );
 };
+
+export const FeatureTTSDropdownInput = (props: FeatureDropdownInputProps) => {
```

Comment:
This should not be in base.tsx because it's not a base feature. Keep it to the text to speech preferences file.

Reply:
Then you won't need to mangle the type either

Reply:
goofmoment